setup.cfg

```diff
@@ -1,7 +1,6 @@
 [metadata]
 name = pyerfa
-author = The Astropy Developers
-author_email = astropy.team@gmail.com
+author = The PyERFA Developers
```

Comment:
Probably we shall also update the LICENSE.rst file accordingly

Reply:
Ah, good point! now done.